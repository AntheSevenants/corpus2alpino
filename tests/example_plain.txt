Dit is een test om te kijken
of gewone tekst
op een gewone manier
geparst kan worden
