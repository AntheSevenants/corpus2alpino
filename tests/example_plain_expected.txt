0-0|Dit is een test om te kijken

1-0|of gewone tekst

2-0|op een gewone manier

3-0|geparst kan worden

